app/2.0.x/index.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: Documentation for Kong Gateway
-is_homepage: true
+title: Documentation for {{site.ce_product_name}}
+subtitle: A guide for {{site.ce_product_name}}
```

Comment:
Not sure that we really have anything to put in the subtitle field. Let's just remove it from the Kong Gateway and Kong Enterprise landing pages.